WcaOnRails/app/views/delegate_reports/_delegate_report.html.erb

```diff
@@ -13,7 +13,7 @@
 
   <div>
     <strong>Location</strong>
-    <%= competition.cityName %>, <%= competition.country.name_in(:en) %>
+    <%= competition.cityName %>, <%= competition.country.name %>
```

Comment:
I didn't this one the first time :(
That's the same idea: this page is not localized yet, so when it's rendered outside a "with_locale" the country name will be localized, which I think doesn't belong to this PR.